examples/src/main/java/org/apache/ignite/examples/datagrid/CacheContinuousQueryWithTransformerExample.java

```diff
@@ -0,0 +1,151 @@
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.examples.datagrid;
+
+import java.util.Arrays;
+import javax.cache.Cache;
+import javax.cache.configuration.Factory;
+import javax.cache.configuration.FactoryBuilder;
+import javax.cache.event.CacheEntryEvent;
+import org.apache.ignite.Ignite;
+import org.apache.ignite.IgniteCache;
+import org.apache.ignite.Ignition;
+import org.apache.ignite.cache.query.ContinuousQueryWithTransformer;
+import org.apache.ignite.cache.query.QueryCursor;
+import org.apache.ignite.examples.ExampleNodeStartup;
+import org.apache.ignite.examples.model.Address;
+import org.apache.ignite.examples.model.Employee;
+import org.apache.ignite.examples.model.EmployeeKey;
+import org.apache.ignite.lang.IgniteClosure;
+
+/**
+ * This examples demonstrates continuous query with transformer API.
+ * <p>
+ * Remote nodes should always be started with special configuration file which
+ * enables P2P class loading: {@code 'ignite.{sh|bat} examples/config/example-ignite.xml'}.
+ * <p>
+ * Alternatively you can run {@link ExampleNodeStartup} in another JVM which will
+ * start node with {@code examples/config/example-ignite.xml} configuration.
+ *
+ * @see CacheContinuousQueryExample
+ * @see CacheContinuousAsyncQueryExample
+ * @see ContinuousQueryWithTransformer
+ */
+public class CacheContinuousQueryWithTransformerExample {
+    /** Cache name. */
+    private static final String CACHE_NAME = CacheContinuousQueryExample.class.getSimpleName();
+
+    public static void main(String[] args) throws Exception {
+        try (Ignite ignite = Ignition.start("examples/config/example-ignite.xml")) {
+            System.out.println();
+            System.out.println(">>> Cache continuous query with transformer example started.");
+
+            // Auto-close cache at the end of the example.
+            try (IgniteCache<EmployeeKey, Employee> cache = ignite.getOrCreateCache(CACHE_NAME)) {
+                // Create new continuous query with transformer.
+                ContinuousQueryWithTransformer<EmployeeKey, Employee, String> qry =
+                    new ContinuousQueryWithTransformer<>();
+
+                //Factory to create transformers.
+                Factory<IgniteClosure<CacheEntryEvent<? extends EmployeeKey, ? extends Employee>, String>> factory =
+                    FactoryBuilder.factoryOf(
+                        //Return one field of complex object.
+                        //Only returned field will be send to local node over network.
```

Comment:
"Only returned field will be send to local node over network." -> Only this field will be sent over to a local node over the network.

Reply:
fixed